just_audio/android/src/main/java/com/ryanheise/just_audio/AudioPlayer.java

```diff
@@ -313,6 +323,14 @@ public void onMethodCall(final MethodCall call, final Result result) {
                 setSpeed((float) ((double) ((Double) request.get("speed"))));
                 result.success(new HashMap<String, Object>());
                 break;
+            case "setSkipSilence":
+                setSkipSilence((Boolean) request.get("enabled"));
+                result.success(null);
```

Comment:
Following the style of the rest of the file, these should return a JSON-encoded instance of the appropriate return type, in this case `SetSkipSilenceResponse`. The same applies to `setVolumeBoost` below.